SaintCoinach/SaintCoinach.csproj

```diff
@@ -62,8 +62,14 @@
       <HintPath>$(SolutionDir)packages\System.Data.SQLite.Linq.1.0.105.2\lib\net451\System.Data.SQLite.Linq.dll</HintPath>
     </Reference>
     <Reference Include="System.Drawing" />
+    <Reference Include="System.Runtime.CompilerServices.Unsafe, Version=4.0.6.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
Why are you including these explicitly via Hint Paths?

Reply:
They are part of a program I'm using the library for. They can be left out.